src/AssemblyAI/Realtime/Types/TerminateSession.cs

```diff
@@ -4,11 +4,11 @@
 
 namespace AssemblyAI;
 
-public class TerminateSession
+public record TerminateSession
 {
     /// <summary>
     /// Set to true to end your streaming session forever
     /// </summary>
     [JsonPropertyName("terminate_session")]
-    public bool TerminateSession_ { get; init; }
+    public required bool TerminateSession_ { get; init; }
```

Comment:
Why is this suffixed with `_`?

Reply:
The compiler complains if a property has the same name as the class

Reply:
This is fine. We won't actually expose this class to users.
The realtime bits are mostly handwritten.